.github/labeler.yml

```diff
@@ -0,0 +1,50 @@
+# Folder Rules
+'Changelog Required':
+- '!html/changelogs/*.yaml'
+
+Database:
+- SQL/*
+
+'🗺️ Mapping - Aurora':
+- maps/aurora/*
+
+'🗺️ Mapping - Away Ship/Away Site':
+- maps/away/*
+
+'🗺️Mapping - Dungeon':
+- maps/dungeon_spawns/*
+
+'🗺️Mapping - Event':
+- maps/event/*
+
+'🗺️Mapping - Random Ruins':
+- maps/random_ruins/*
+
+'🗺️Mapping - Runtime':
+- maps/runtime/*
+
+'🗺️Mapping - Horizon':
+- maps/sccv_horizon/*
+
+'🗺️Mapping - Templates':
```

Comment:
```suggestion
'🗺️ Mapping - Dungeon':
- maps/dungeon_spawns/*

'🗺️ Mapping - Event':
- maps/event/*

'🗺️ Mapping - Ruin':
- maps/random_ruins/*

'🗺️ Mapping - Runtime':
- maps/runtime/*

'🗺️ Mapping - Horizon':
- maps/sccv_horizon/*

'🗺️ Mapping - Templates':
```

Reply:
Thats not how the labels are currently set up